packages/sanity/src/core/presence/FieldPresence.tsx

```diff
@@ -60,35 +56,30 @@ export const FieldPresenceInner = memo(function FieldPresenceInner({
       : null,
   ].filter(Boolean)
 
-  const maxWidth = calcAvatarStackWidth(maxAvatars)
-  const currWidth = Math.min(calcAvatarStackWidth(uniquePresence.length), maxWidth)
-
   return (
-    <FlexWrapper justify="flex-end" style={{width: maxWidth}}>
+    <FlexWrapper>
       <div />
 
-      <PresenceTooltip items={uniquePresence}>
-        <InnerBox direction="row-reverse" style={{width: currWidth}}>
-          {avatars.map(
-            (av, i) =>
-              av && (
-                <div
-                  key={av.key}
-                  style={{
-                    position: 'absolute',
-                    transform: `translate3d(${-i * (AVATAR_SIZE + AVATAR_DISTANCE)}px, 0px, 0px)`,
-                    transitionProperty: 'transform',
-                    transitionDuration: '200ms',
-                    transitionTimingFunction: 'cubic-bezier(0.85, 0, 0.15, 1)',
-                    zIndex: 100 - i,
-                  }}
-                >
-                  {av.element}
-                </div>
-              )
-          )}
-        </InnerBox>
-      </PresenceTooltip>
+      <InnerBox direction="row-reverse">
```

Comment:
**question:** really minor, but was there a reason for disabling presence tooltips when `DISABLE_OVERLAY = true`?

Reply:
I had to move the tooltip to `FieldPresenceWithOverlay` to make it work together with `onMouseEnter` and `onMouseLeave` interactions to display field actions.